tests/topotests/lib/bgp.py

```diff
@@ -343,6 +344,12 @@ def __create_bgp_global(tgen, input_dict, router, build=False):
 
                 config_data.append(cmd)
 
+    bgp_always_compare_med = bgp_data.setdefault("bgp_always_compare_med", None)
```

Comment:
Why can't we set it to False by default? None sure why None/Bool is mixed here.

Reply:
Updated code.